### PR TITLE
Emphasize UTF-8 encoding for configs on Windows

### PR DESCRIPTION
**Pull request by gcymbalski**
Addresses sensu/sensu#1232

### CODE REVIEW

**Comment by gcymbalski** [comment]
Urgh, Hub isn't respecting my configs. One more try.

**Comment by cwjohnston** [comment]
My understanding of this issue is that Notepad.exe and possibly other common Windows editors add Byte Order Marking to UTF-8 which we can't parse. Seems to me this is more likely to occur via manual editing instead of programatic generation of JSON.

I think we need to explicitly describe UTF-8 BOM as different from plain UTF-8, and, if possible, link to some reasonably authoritative reference on the matter.

http://stackoverflow.com/questions/2223882/whats-different-between-utf-8-and-utf-8-without-bom

File: docs/0.28/platforms/sensu-on-microsoft-windows.md [hunk 1]
```diff
@@ -202,3 +205,5 @@ To manually start and stop the Sensu client Windows service, use the
 [13]: #example-transport-configuration
 [14]: #example-client-configuration
 [15]: ../reference/transport.html#transport-definition-specification
+[16]: http://stackoverflow.com/questions/2223882/whats-different-between-utf-8-and-utf-8-without-bomhttp://stackoverflow.com/questions/2223882/whats-different-between-utf-8-and-utf-8-without-bom
```
Comment (cwjohnston) [inline docs/0.28/platforms/sensu-on-microsoft-windows.md]: link is doubled up here

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Closes https://github.com/sensu/sensu/issues/1232

I think this is really close, just one fix needed.

**Comment by gcymbalski** [comment]
Oof, that was dumb. Resolved, thanks!

**Review by cwjohnston — APPROVED** [review]

**Pull request merged**